.travis.yml

```diff
@@ -0,0 +1,29 @@
+# This config file for Travis CI
+sudo: required
+dist: trusty
+services:
+  - docker
+language: cpp
```

Comment:
not sure this is needed.

Reply:
This enables the `cache: ccache` shorthand, but does not have any other benifits (for use with `industrial_ci`).